### ISSUE TITLE
E741 ambiguous variable name 'l' between flake8 and pycodesyle

### ISSUE THREAD

**Issue by RaminFP** [PyCQA/pycodestyle#598]
Hi,
i have question ,

i test this on pycodestyle 

```python 
l=0
```
output : `E741 ambiguous variable name 'l' `

but flake8 is wrapper of pycodestyle,... not found this, E741 why?

**Comment by spaceone** [comment]
because flake8 doesn't use pycodestyle 2.1.0 yet.

**Comment by sigmavirus24** [comment]
@spaceone is correct.

**Comment by RaminFP** [comment]
@spaceone  thanks,this means flake8 update pycodestyle 2.1.0 ?

**Comment by spaceone** [comment]
@RaminFP Yes, flake8 will have to update its pycodestyle-compatibility. This is planned but probably takes a few time: See https://gitlab.com/pycqa/flake8/issues/246 https://github.com/PyCQA/flake8/pull/7

**Comment by RaminFP** [comment]
Thanks again,

**Comment by RaminFP** [comment]
@spaceone sorry again for question!there is this warning E741 ambiguous variable name 'l' of pycodestyle on new version flake8?

**Comment by dhood** [comment]
@raminfp FYI flake8 3.5.0 has E741 added https://gitlab.com/pycqa/flake8/commit/ee435593488a5c3b9805bc686b83d17505491c42
